projects/algorai-finance/constants.js

```diff
@@ -63,6 +63,20 @@ const vaults = [
     },
 ];
 
+const state = {
+    ethereum: {
+        contractAddress: "0x2103dE019522B65BbbD051C8Ce9426AA9005A655",
```

Comment:
this looks like a wallet rather than a smart contract, can you give some context why this is count as tvl? most cases we dont count wallet balances as tvl